src/main/java/com/teamcautionrobotics/robot/Robot.java

```diff
@@ -125,8 +128,20 @@ public void robotInit() {
      */
     @Override
     public void robotPeriodic() {
+        boolean precisionTurningEngaged = driverRight.getTrigger();
+        if (precisionTurningEngaged) {
+            driverLeft.setDeadband(driverLeft.getDeadband() * PRECISION_TURNING_SCALING_FACTOR);
+        } else {
+            driverLeft.setDeadband(0.1);
```

Comment:
Magic number alert!
Consider moving the 0.1 into a variable (probably a constant) for the default turning deadband.